source/unified-test-format/unified-test-format.rst

```diff
@@ -2649,6 +2650,14 @@ to elsewhere in the specification as the internal MongoClient.
 Determine the server version and topology type using the internal MongoClient.
 This information will be used to evaluate any future `runOnRequirement`_ checks.
 
+In addition to the cluster URI, the test runner MUST also be configurable
```

Comment:
I think this may be the only reference to "cluster URI". Can we change this to "aforementioned connection string" so it's clear we're referring back to the thing mentioned in the first paragraph of this section?

Reply:
Done. The change looks much bigger than it is due to editor line wrapping at 80 characters.